openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -78,213 +142,441 @@
-#
-#        https://docs.openshift.com/container-platform/3.3/install_config/
-#        aggregate_logging.html
+        A control response is planned. Engineering progress can be tracked via:
 
+        https://issues.redhat.com/browse/CMP-227
     - key: b
       text: |
-        'OpenShift logging subsystems do not alter the original contents
-        or time ordering of audit records.'
+        A control response is planned. Engineering progress can be tracked via:
 
-#       ADMINISTRATORS NOTE FOR AU-7(b):
-#       OpenShift can be configured to rely upon the underlying audit subsystem
-#       of Red Hat Enterprise Linux. This subsystem has undergone Common
-#       Criteria certification testing which verified log integrity during
-#       remote transmittal. Certification documents can be found publicly at:
-#
-#       https://www.redhat.com/en/technologies/industries/government/standards
+        https://issues.redhat.com/browse/CMP-227
 
 - control_key: AU-7 (1)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: planned
   narrative:
     - text: |
-        'Customers are responsible for ensuring that the information system
-        provides the capability to process audit records for events of interest
-        based on defined fields within audit records. Those fields are generally
-        defined at an organizational level through AU-3.  A successful control
-        response will discuss how audit records can be queried based on these
-        fields.'
+        A control response is planned. Engineering progress can be tracked via:
+
+        https://issues.redhat.com/browse/CMP-228
+
+- control_key: AU-7 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: partial
+  narrative:
+    - text: |
+        A control response is planned. Engineering progress can be tracked via:
+
+        https://issues.redhat.com/browse/CMP-229
 
 - control_key: AU-8
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: planned
   narrative:
     - key: a
       text: |
-        'OpenShift logs are generated using the internal system clock by
-        default. This is non-configurable behavior.'
+        A control response is planned. Engineering progress can be tracked via:
 
+        https://issues.redhat.com/browse/CMP-230
     - key: b
       text: |
-        'OpenShift time keeping relies on the time services provided by the
-        underlying operating system. When running on Red Hat Enterprise Linux,
-        the system time is always kept in Coordinated Universal Time (UTC) and
-        converted in applications to local time as needed. Local time is the
-        actual time on your current time zone, taking into account daylight
-        savings time.
-
-        OpenShift hosts are configured to use UTC by default. To manually
-        verify this, the timedatectl command may be used:
-
-        $ timedatectl set-local-rtc no
```

Comment:
Many existing control responses have been removed, such as this one, which appears to be valid. How come?

Reply:
@shawndwells these have been removed because the existing control responses are under OpenShift 3 assumptions which maybe (and probably) are no longer valid in OpenShift 4. For example, the Machine Config operator may be what sets time/date vs running a command on RHEL node. These responses very well might come back, but we shouldn't keep them if they are wrong. Also, we are tracking control responses in JIRA, so it's not like we will drop the ball or lose the bubble on these responses.